docs/pages/includes/metrics.mdx

```diff
@@ -1,59 +1,67 @@
+| `gcs_event_storage_downloads_seconds`                    | histogram | GCP GCS             | Latency for GCS download operations.                                                               |
+| `gcs_event_storage_downloads`                            | counter   | GCP GCS             | Number of downloads from the GCS backend.                                                          |
+| `gcs_event_storage_uploads_seconds`                      | histogram | GCP GCS             | Latency for GCS upload operations.                                                                 |
+| `gcs_event_storage_uploads`                              | counter   | GCP GCS             | Number of uploads to the GCS backend.                                                              |
+| `grpc_server_started_total`                              | counter   | Teleport Auth       | Total number of RPCs started on the server.                                                        |
+| `grpc_server_handled_total`                              | counter   | Teleport Auth       | Total number of RPCs completed on the server, regardless of success or failure.                    |
+| `grpc_server_msg_received_total`                         | counter   | Teleport Auth       | Total number of RPC stream messages received on the server.                                        |
+| `grpc_server_msg_sent_total`                             | counter   | Teleport Auth       | Total number of gRPC stream messages sent by the server.                                           |
+| `heartbeat_connections_received_total`                   | counter   | Teleport Auth       | Number of times the Auth Service received a heartbeat connection.                                  |
+| `s3_requests_total`                                      | counter   | Amazon S3           | Total number of requests to the S3 API.                                                            |
+| `s3_requests`                                            | counter   | Amazon S3           | Total number of requests to the S3 API grouped by result.                                          |
+| `s3_requests_seconds`                                    | histogram | Amazon S3           | Request latency for the S3 API.                                                                    |
+| `teleport_audit_emit_events`                             | counter   | Teleport Audit Log  | Number of audit events emitted.                                                                    |
+| `teleport_audit_parquetlog_batch_processing_seconds`     | histogram | Teleport Audit Log  | Duration of processing single batch of events in parquet audit log.                                |
+| `teleport_audit_parquetlog_s3_flush_seconds`             | histogram | Teleport Audit Log  | Duration of flushing parquet files to s3 in parquet audit log.                                     |
```

Comment:
We should capitalize third-party product names as they are capitalized in their docs. This means using "Parquet", "S3", etc.